### ISSUE TITLE
UI/UX: GaussianNB Model Parameters

### ISSUE THREAD

**Issue by Avanznow** [Avanznow/Tellekynezix#10]
# 🚀 Dynamic Parameters UI for GaussianNB Model

### **Description**
Excellent progress, team! The **Avatar GUI** is now successfully running on Pop!_OS. Our next milestone is to make the **Parameters** panel functional. 

Currently, when a user selects **GaussianNB**, the right-hand panel remains empty. We need to implement a dynamic interface that empowers the Lead PI and researchers to fine-tune model hyperparameters directly from the GUI before initiating the training process. 🛠️

### **Proposed Requirements** 📋
When the **GaussianNB** model is active, the UI must expose the following hyperparameters:

* **Var Smoothing:** A numeric input to control the portion of the largest variance of all features that is added to variances for calculation stability.
    * **Default:** $1e-9$
    * **Range:** $1e-12$ to $1e-1$
* **Priors:** A field to set custom class prior probabilities (defaults to `None`).
* **Data Integration:** These values must be captured and passed to the `BrainFlowDataProcessor` backend before the **Train** 🟡 process begins.
* **Feedback:** The Console Log should confirm when a parameter is updated (e.g., *"Var Smoothing set to 1e-8"*).

### **Technical Implementation Blueprint** 💻
To ensure a clean and scalable architecture, we will implement a dynamic layout update within `GUI5.py`. The following function rascals how to clear and rebuild the **Parameters** container using **PySide6**:

```python
from PySide6.QtWidgets import QVBoxLayout, QLabel, QDoubleSpinBox, QLineEdit, QFormLayout, QWidget

def update_params_panel(self, model_name):
    """
    Dynamically updates the right-hand Parameters panel based on the selected model.
    """
    # 1. Clear existing widgets in the parameters layout
    while self.params_layout.count():
        item = self.params_layout.takeAt(0)
        widget = item.widget()
        if widget:
            widget.deleteLater()

    # 2. Add Header for the selected model
    header = QLabel(f"Settings: {model_name}")
    header.setStyleSheet("font-weight: bold; font-size: 14px; color: #FFFFFF; margin-bottom: 10px;")
    self.params_layout.addWidget(header)

    # 3. Build GaussianNB specific form
    if model_name == "GaussianNB":
        form = QFormLayout()
        
        # Var Smoothing with Scientific Notation support
        self.var_smoothing_input = QDoubleSpinBox()
        self.var_smoothing_input.setRange(1e-12, 0.1)
        self.var_smoothing_input.setDecimals(12)
        self.var_smoothing_input.setValue(1e-9)
        self.var_smoothing_input.setSingleStep(1e-10)
        
        # Priors Input
        self.priors_input = QLineEdit()
        self.priors_input.setPlaceholderText("None (Default)")
        
        form.addRow(QLabel("Var Smoothing:"), self.var_smoothing_input)
        form.addRow(QLabel("Priors:"), self.priors_input)
        
        container = QWidget()
        container.setLayout(form)
        self.params_layout.addWidget(container)
        
    # 4. Placeholder for future Deep Learning frameworks (PyTorch/JAX/TF)
    elif model_name in ["PyTorch", "TensorFlow", "JAX"]:
        self.params_layout.addWidget(QLabel("Learning Rate: [ 0.001 ]"))
        self.params_layout.addWidget(QLabel("Batch Size: [ 32 ]"))

    self.params_layout.addStretch()
```

### **Why this is critical?** 🌟
This is a high-priority update for **Project Nexus**. Precision in Gaussian classification is key to the stability of our **Neurohaptics Telekinesis Avatars**. Let's make this interface as powerful and flexible as the hardware it controls! ⚡

---
**Labels:** `enhancement` `UI/UX` `priority-high` `Nexus-Project`
